.github/workflows/comment-on-asciidoc-changes.yml

```diff
@@ -2,7 +2,8 @@
 name: Comment on PR for .asciidoc changes
 
 on:
-  pull_request:
+  # We need to use pull_request_target to be able to comment on PRs from forks
+  pull_request_target:
```

Comment:
Given the `comment-on-asciidoc-changes` workflow checks out the code from a PR, allowing the action to run from forks is inherently risky. While I dont see an injection point for execution of malicious code I think it is a significant surface area to try to keep safe. Would it be possible to instead use the github API to detect if files are changed (Rather than checking out the code in the action context? https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#list-pull-requests-files )

Reply:
Thank you for the feedback.

I just found that the action in use can utilize the GitHub API instead of git.

https://github.com/tj-actions/changed-files?tab=readme-ov-file#using-githubs-api-octocat

I will verify if this works as expected for forks.